components/automate-cli/cmd/chef-automate/config.go

```diff
@@ -19,15 +23,47 @@ var configCmdFlags = struct {
 	overwriteFile bool
 	timeout       int64
 	acceptMLSA    bool
+
+	// automate    bool
+	// chef_server bool
+	frontend   bool
+	opensearch bool
+	postgresql bool
 }{}
 
+const (
+	FRONTEND_COMMANDS = `
+	sudo chef-automate config patch /tmp/%s;
+	export TIMESTAMP=$(date +'%s');
+	sudo mv /etc/chef-automate/config.toml /etc/chef-automate/config.toml.$TIMESTAMP;
+	sudo chef-automate config show > sudo /etc/chef-automate/config.toml`
+
+	BACKEND_COMMAND = `
+	export HAB_LICENSE=accept-no-persist;
+	export TIMESTAMP=$(date +"%s");
+	source /hab/sup/default/SystemdEnvironmentFile.sh;
+	automate-backend-ctl applied --svc=automate-ha-%s | tail -n +2 > /tmp/pg_config.toml; 
+	cat /tmp/pg_config.toml >> /tmp/%s ;
+	sudo hab config apply automate-ha-postgresql.default  $(date '+%s') /tmp/pg_config.toml;   
+	`
+
+	dateFormat = "%Y%m%d%H%M%S"
+)
+
 func init() {
 	configCmd.AddCommand(showConfigCmd)
 	configCmd.AddCommand(patchConfigCmd)
 	configCmd.AddCommand(setConfigCmd)
 
 	showConfigCmd.Flags().BoolVarP(&configCmdFlags.overwriteFile, "overwrite", "o", false, "Overwrite existing config.toml")
 
+	// patchConfigCmd.PersistentFlags().BoolVarP(&configCmdFlags.automate, "automate", "a", false, "Patch toml configuration to the automate node")
+	// patchConfigCmd.PersistentFlags().BoolVarP(&configCmdFlags.chef_server, "chef_server", "c", false, "Patch toml configuration to the chef_server node")
+	patchConfigCmd.PersistentFlags().BoolVar(&configCmdFlags.frontend, "frontend", false, "Patch toml configuration to the all frontend nodes")
+	patchConfigCmd.PersistentFlags().BoolVar(&configCmdFlags.frontend, "fe", false, "Patch toml configuration to the all frontend nodes[DUPLICATE]")
```

Comment:
Why duplicate flag?

Reply:
To enable flag availability for shorthand with 2 characters (--fe)